The new ffmpeg-1.0 comes with the headers libavutil/time.h and 
libavutil/log.h.  These overrule /usr/include/time.h and 
minidlna-1.0.25/log.h due to the priority of '-I' set in the
Makefile

CFLAGS = -Wall -g -O3 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
	 -I/usr/include/ffmpeg \
	 -I/usr/include/libavutil -I/usr/include/libavcodec -I/usr/include/libavformat \
	 -I/usr/include/ffmpeg/libavutil -I/usr/include/ffmpeg/libavcodec -I/usr/include/ffmpeg/libavformat

Here below is a patch removing the need for these search
directories.  Backward compatibility could be guaranteed via
'-I/usr/include/ffmpeg'.

diff -u -p -r minidlna-1.0.25/Makefile minidlna-1.0.25-new/Makefile
--- minidlna-1.0.25/Makefile	2012-01-17 23:49:01.000000000 +0100
+++ minidlna-1.0.25-new/Makefile	2012-10-25 01:53:04.000000000 +0200
@@ -12,10 +12,14 @@
 #
 #CFLAGS = -Wall -O -D_GNU_SOURCE -g -DDEBUG
 #CFLAGS = -Wall -g -Os -D_GNU_SOURCE
-CFLAGS = -Wall -g -O3 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
-	 -I/usr/include/ffmpeg \
-	 -I/usr/include/libavutil -I/usr/include/libavcodec -I/usr/include/libavformat \
-	 -I/usr/include/ffmpeg/libavutil -I/usr/include/ffmpeg/libavcodec -I/usr/include/ffmpeg/libavformat
+CFLAGS = -Wall -g -O3 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -I/usr/include/ffmpeg
+#	 -I/usr/include/libavcodec
+#	 -I/usr/include/libavformat
+# overrules /usr/include/time.h
+#	 -I/usr/include/libavutil 
+# nonexistant
+#	 -I/usr/include/ffmpeg \
+#	 -I/usr/include/ffmpeg/libavutil -I/usr/include/ffmpeg/libavcodec -I/usr/include/ffmpeg/libavformat
 #STATIC_LINKING: CFLAGS += -DSTATIC
 #STATIC_LINKING: LDFLAGS = -static
 CC = gcc
diff -u -p -r minidlna-1.0.25/metadata.c minidlna-1.0.25-new/metadata.c
--- minidlna-1.0.25/metadata.c	2012-06-29 23:11:29.000000000 +0200
+++ minidlna-1.0.25-new/metadata.c	2012-10-25 01:53:37.000000000 +0200
@@ -30,9 +30,9 @@
 #include "image_utils.h"
 #include <jpeglib.h>
 #include <setjmp.h>
-#include <avutil.h>
-#include <avcodec.h>
-#include <avformat.h>
+#include <libavutil/avutil.h>
+#include <libavcodec/avcodec.h>
+#include <libavformat/avformat.h>
 #include "tagutils/tagutils.h"
 
 #include "upnpglobalvars.h"
diff -u -p -r minidlna-1.0.25/tagutils/tagutils-plist.c minidlna-1.0.25-new/tagutils/tagutils-plist.c
--- minidlna-1.0.25/tagutils/tagutils-plist.c	2012-01-17 23:54:47.000000000 +0100
+++ minidlna-1.0.25-new/tagutils/tagutils-plist.c	2012-10-25 01:49:33.000000000 +0200
@@ -27,7 +27,7 @@
 #include "misc.h"
 #include "tagutils.h"
 #include "textutils.h"
-#include "log.h"
+#include "../log.h"
 
 
 #define MAX_BUF 4096
